### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Eti-Fromentin**
A misplaced br tag may or may not be hidden here. 
Do you have the courage to find the Schrodinger br? 

-

### CODE REVIEW

**Review by sophielepert — DISMISSED** [review]

**Review by Vinz19 — DISMISSED** [review]

File: src/Components/Age.jsx [hunk 1]
```diff
@@ -25,15 +25,19 @@ function Age() {
         <p className="BirthDateCheckBox">Sorry, but we have to ask : </p>
         <h3>Are you 18 years old ?</h3>
         <p className="BirthDateCheckBox">
-          <button className="AgeButton">Yes</button>
+          <Link to="./HomeConditions">
```
Comment (pelegrand) [inline src/Components/Age.jsx:28]: remove Links because useHistory is enough

File: src/Components/RecipeCard.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import More from '../Assets/More.png';
```
Comment (pelegrand) [inline src/Components/RecipeCard.jsx:2]: the order of components is right just add space line between each block plz

File: src/Components/Slider.jsx [hunk 3]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Card from './Card';
+
+// Import Swiper React components
```
Comment (pelegrand) [inline src/Components/Slider.jsx:4]: remove all comments

File: src/Components/Slider.jsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import 'swiper/css/pagination';
+import './Slider.css';
+
+// import Swiper core and required modules
+import SwiperCore, { Pagination, Navigation } from 'swiper';
+
+// install Swiper modules
+SwiperCore.use([Pagination, Navigation]);
+
+export default function App() {
```
Comment (pelegrand) [inline src/Components/Slider.jsx:19]: use the same architecture for each component

**Review by pelegrand — CHANGES_REQUESTED** [review]

**Review by pelegrand — APPROVED** [review]

**Pull request merged**